statistics/boostrap.go

```diff
@@ -124,7 +150,7 @@ func (h *Handle) initStatsBuckets(tables statsCache) error {
 				continue
 			}
 			hist = &index.Histogram
-			lower, upper = row.GetDatum(6, &fields[6].Column.FieldType), row.GetDatum(7, &fields[7].Column.FieldType)
+			lower, upper = types.NewBytesDatum(row.GetBytes(6)), types.NewBytesDatum(row.GetBytes(7))
```

Comment:
The memory is a reference to the Chunk, and the Chunk will be reset and reused for next bytes.